week-4/E-arrays-of-objects/exercise-3.js

```diff
@@ -59,12 +59,26 @@ var restaurantFinderApplication = {
   applicationVersion: "1.0",
   restaurants: restaurants,
   findAvailableRestaurants: function(numberOfPeople) {
+    return restaurants
+      .filter(
+        restaurant =>
+          restaurant.totalSeats -
+            restaurant.numberOfCustomers -
+            numberOfPeople >=
+          0
+      )
+      .map(restaurant => restaurant.name);
     // Complete here
```

Comment:
Try to get in the habit of removing any outdated comments when you commit. You could remove the `// Complete here` comments when you've done the work.

Reply:
Thanks. I will